Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
I thought we were using ko to assemble the application, hence we don't need a Dockerfile here.

Reply:
that makes sense, i'll update workflow to run ko and remove the docker file